src/connector.js

```diff
@@ -1,5 +1,6 @@
 const net = require('net');
 const dns = require('dns');
+const punycode = require('punycode');
```

Comment:
The `punycode` module that ships with Node.js has been deprecated, and it is recommended to use https://github.com/bestiejs/punycode.js instead. See the comment over at https://nodejs.org/api/punycode.html#punycode_punycode.

Reply:
🙇‍♀️ Thanks for pointing that out!